rubbery-raccoons/wired/wired_app/models.py

```diff
@@ -1,5 +1,5 @@
-from django.db import models
 from django.contrib.auth import get_user_model
+from django.db import models
 
 
 class Article(models.Model):
```

Comment:
Not sure if this needs to be in a different section but since this hasnt been merged yet i noticed something here

`pubication_date = DatetimeField()`

If we decide to use articles that are not generated at the time of writing, meaning not using datetime.now() it seems that this would cause issues because its a niave time object. Should we change this to just a DateField object in stead.

I noticed this trying to migrate data into the db.